app/models/movie.rb

```diff
@@ -0,0 +1,12 @@
+class Movie < ApplicationRecord
+  validates :title, presence: true, uniqueness: true
+  validates :overview, presence: true
+  validates :release_date, presence: true
+  validates :inventory, presence: true, numericality: { greater_than: -1, only_integer: true }
```

Comment:
yeah! like, here where you check that the inventory is an integer or that movie names are unique. That's what I was thinking in my previous comment 🙂